docs/source/examples/ddd_on_ibmq_ghz.md

```diff
@@ -0,0 +1,243 @@
+            circuit_to_run = convert_from_mitiq(noisy, circuit_type)
+            job = qiskit.execute(
+                experiments=circuit_to_run,
+                backend=qiskit.Aer.get_backend("qasm_simulator"),
+                optimization_level=0,
+                shots=shots,
+            )
+            all_counts = job.result().get_counts()
+            P0 = (
+                all_counts.get(
+                    ("".join(map(str, correct_bitstring))).replace("", " ")[1:-1], 0.0
+                )
+                / shots
+            )
+            return P0
+
+        else:
+            job = qiskit.execute(
+                experiments=circuit_to_run,
+                backend=qiskit.Aer.get_backend("qasm_simulator"),
+                optimization_level=0,
+                shots=shots,
+            )
+
+    # Convert from raw measurement counts to the expectation value
+    all_counts = job.result().get_counts()
+    P0 = all_counts.get("".join(map(str, correct_bitstring)), 0.0) / shots
+    return P0
+```
+
+## Select the DDD sequences to be applied
+We now import a DDD _rule_ from Mitiq, i. e., a function that generates DDD sequences of different length.
+In this example, we opt for YY sequences (pairs of Pauli Y operations).
+
+```{code-cell} ipython3
+from mitiq import ddd
+
+rule = ddd.rules.yy
+```
+
+## Sample bitstrings from GHZ circuits
+
+```{code-cell} ipython3
+true_values, noisy_values, ddd_values = [], [], []
+
+for nq in num_qubits:
+    true_nqubits_values, noisy_nqubits_values, ddd_nqubits_values = [], [], []
+    circuit, correct_bitstring = get_circuit(num_qubits=nq)
+
+    for trial in range(trials):
+
+        true_nqubits_values.append(
+            ibmq_executor(circuit, shots, correct_bitstring, noisy_sim=False)
+        )
+
+        noisy_nqubits_values.append(
+            ibmq_executor(circuit, shots, correct_bitstring, noisy_sim=True)
+        )
+
+        noisy_executor = functools.partial(
+            ibmq_executor,
+            shots=shots,
+            correct_bitstring=correct_bitstring,
+        )
+
+        ddd_nqubits_values.append(
+            ddd.execute_with_ddd(
+                circuit,
+                noisy_executor,
+                rule=rule,
+            )
+        )
+
+    true_values.append(true_nqubits_values)
+    noisy_values.append(noisy_nqubits_values)
+    ddd_values.append(ddd_nqubits_values)
+```
+
+Now we can visualize the results.
+
+```{code-cell} ipython3
+avg_true_values = np.average(true_values, axis=1)
+avg_noisy_values = np.average(noisy_values, axis=1)
+
+std_true_values = np.std(true_values, axis=1, ddof=1)
+std_noisy_values = np.std(noisy_values, axis=1, ddof=1)
+
+avg_ddd_values = np.average(ddd_values, axis=1)
+std_ddd_values = np.std(ddd_values, axis=1, ddof=1)
+
+plt.figure(figsize=(9, 5))
+
+plt.plot(num_qubits, avg_true_values, "--", label="True", lw=2)
+eb = plt.errorbar(
+    num_qubits, avg_noisy_values, yerr=std_noisy_values, label="Raw", ls="-."
+)
```

Comment:
Error bars are so small that are practically invisible. 
This is probably due to GHZ circuits being deterministic as opposed to rb/mirror. 

Possible solutions:

- Reducing number of shots to increase error bars.
- Avoiding the loop over multiple trials and the error bars.

Reply:
Solved by reducing shots down to 1000.